include/alpaka/wait/Traits.hpp

```diff
@@ -17,11 +17,11 @@ namespace alpaka
     namespace trait
     {
         //! The thread wait trait.
-        template<typename TAwaited, typename TSfinae = void>
```

Comment:
this should not be part of the PR and is not required. The sfinae param ,should not be removed.

Reply:
You are right, this sneaked in from some other branch by mistake